agent/agent_test.go

```diff
@@ -135,7 +135,7 @@ func TestRunnerRun(t *testing.T) {
 			t.Error(err.Error())
 		}
 		monitor := new(mockMonitor)
-		monitor.On("RecordRun", validRecipe, mock.AnythingOfType("int"), true).Once()
+		monitor.On("RecordRun", validRecipe, mock.AnythingOfType("int"), true, 0).Once()
```

Comment:
try to use other number than `0`, this is to make sure we do not test default values.